### ISSUE TITLE
Splitwise seems to be a better service with similar purpose.

### ISSUE THREAD

**Issue by krist-jin** [neilChenXie/CAC-account-service#1]
No offense but just in case you are interested:
https://www.splitwise.com/

**Comment by neilChenXie** [comment]
@krist-jin  Yes, back that time, I didn't know them and I wrote this for PHP CodeIgniter practice for my boring summertime. 
Thank you for noticing my project. I never expect anyone who will pay attention to it.

**Comment by krist-jin** [comment]
actually I knew this app through Siyao Li when we were in the trip to Vegas a few months ago. Thanks for the app!
